src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -150,6 +158,28 @@ public HttpApiHandler(DurableTaskExtension config, ILogger logger)
             return request.CreateErrorResponse(HttpStatusCode.BadRequest, "No such API");
         }
 
+        private async Task<HttpResponseMessage> HandleGetStatusRequestAsync(
+            HttpRequestMessage request)
+        {
+            try
+            {
+                DurableOrchestrationClientBase client = this.GetClient(request);
+                var status = await client.GetStatusAsync();

```

Comment:
In cases where the type of the variable is not obvious, I prefer we use the actual type name instead of `var`. It makes the code more easily understandable to others. In this case, I would prefer:

```csharp
List<DurableOrchestrationStatus> statusForAllInstances = await client.GetStatusAsync();
```

I apologize that the convention is not consistently used everywhere.

Reply:
Sorry about that. I should have notice that. For `statusForAllInstances` naming is good to know! I was wondering which is the good naming for this. Done!